clouditor-engine-core/build.gradle

```diff
@@ -69,6 +70,10 @@ dependencies {
 
     implementation group: 'org.apache.poi', name: 'poi-ooxml', version: '4.1.2'
 
+    compile group: 'postgresql', name: 'postgresql', version: libraryVersions.postgresql
```

Comment:
You are using an extremely old version of the postgres JDBC driver here (from 2011). The current one is `org.postgresql:postgresql:42.2.18`.